src/main/java/io/github/thebusybiscuit/slimefun4/integrations/OrebfuscatorIntegration.java

```diff
@@ -0,0 +1,64 @@
+package io.github.thebusybiscuit.slimefun4.integrations;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.annotation.Nonnull;
+
+import org.bukkit.Bukkit;
+import org.bukkit.block.Block;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.plugin.RegisteredServiceProvider;
+
+import io.github.thebusybiscuit.slimefun4.api.events.BlockPlacerPlaceEvent;
+import io.github.thebusybiscuit.slimefun4.api.events.ExplosiveToolBreakBlocksEvent;
+import io.github.thebusybiscuit.slimefun4.api.events.ReactorExplodeEvent;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import net.imprex.orebfuscator.api.OrebfuscatorService;
+
+/**
+ * 
+ * This handles block breaks with orebfuscator
+ * 
+ * @author NgLoader
+ *
+ */
+public class OrebfuscatorIntegration implements Listener {
+
+	private final SlimefunPlugin plugin;
+	private OrebfuscatorService service;
+
+	public OrebfuscatorIntegration(@Nonnull SlimefunPlugin plugin) {
+        this.plugin = plugin;
+    }
+
+	/**
+	 * Init orebfuscation service and register listener
+	 */
+    public void register() {
+        RegisteredServiceProvider<OrebfuscatorService> provider = Bukkit.getServer().getServicesManager().getRegistration(OrebfuscatorService.class);
+        this.service = provider.getProvider();
```

Comment:
I don't see a need for declaring a variable here.

```suggestion
        this.service = plugin.getServer().getServicesManager().getRegistration(OrebfuscatorService.class).getProvider();
```

Reply:
It it better to read

Reply:
but when you like it more in a oneline we can change that

Reply:
Not a necessity in my opinion, really comes down to preference I guess.

But once the tab characters are removed I am gonna approve this.
It will likely be up to you or our second reviewer at that point XD
Thanks!

Reply:
Okay I change it 👍🏻